src/foam/blob/BlobBlob.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license
+ * Copyright 2018 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package foam.blob;
+
+import java.nio.ByteBuffer;
+
+public class BlobBlob
```

Comment:
Why is this called a BlobBlob? Wouldn't ByteArrayBlob be a better name?

Reply:
@kgrgreer BlobBlob is what the Javascript equivalent is called

Reply:
@kgrgreer Renamed to ByteArrayBlob